hack/update-helm.sh

```diff
@@ -33,7 +33,7 @@ EXCLUDE_FILES='kustomization.yaml'
 
 # Copy all YAML files from the source directory to the destination directory
 cp ${SRC_CRD_DIR}/*.yaml ${DEST_CRD_DIR}
-cp ${SRC_VISIBILITY_DIR}/*.yaml ${DEST_VISIBILITY_DIR}
+find $SRC_VISIBILITY_DIR -name "*.yaml" $(printf "! -name %s " $EXCLUDE_FILES) -exec cp "{}" $DEST_VISIBILITY_DIR \;
```

Comment:
```suggestion
find ${SRC_VISIBILITY_DIR} -name "*.yaml" $(printf "! -name %s " ${EXCLUDE_FILES}) -exec cp "{}" ${DEST_VISIBILITY_DIR} \;
```

nit

Reply:
[L37](https://github.com/kubernetes-sigs/kueue/blob/main/hack/update-helm.sh#L37)
There was no '{}', so I think it's ok not to add '{}', better keep it same, WDYT?

Reply:
Oh, that makes sense. Thanks.